djconnectwise/models.py

```diff
@@ -406,7 +406,7 @@ def default_fax_number(self):
 class ContactCommunication(models.Model):
     contact = models.ForeignKey(
         'Contact', null=True, on_delete=models.CASCADE)
-    value = models.CharField(blank=True, null=True, max_length=250)
+    value = models.CharField(default='', max_length=250)
```

Comment:
There should be no blank default here, as we want to get rid of anything null or blank. We can add a simple runpython operation in the migration to delete any invalid records (if they exist, but I doubt they do). Let me know if you would like any help with that.